src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs

```diff
@@ -191,12 +191,17 @@ private void addExpressionProfilesUsedBySimulationSubjectToProject(ISimulationSu
             _expressionProfileUpdater.SynchronizeExpressionProfileWithSimulationSubject(expressionProfile, simulationSubject);
 
             //Some parameters are probably marked as FixedValue event thought they have not changed (Formula=>constant) due to change in 
-            //definition of Fraction expressed basolateral going from a constant to a formula. We reset
+            //definition of Fraction expressed basolateral going from a constant to a formula. We reset is fixed value and default state
             expressionProfile.Individual.AllMoleculeParametersFor(expressionProfile.Molecule)
                .Where(x => !x.Visible)
                .Where(x => x.IsFixedValue)
                .Each(x => x.IsFixedValue = false);
 
+            expressionProfile.Individual.AllMoleculeParametersFor(expressionProfile.Molecule)
+               .Where(x => !x.Visible)
+               .Where(x => !x.IsDefault)
+               .Each(x => x.IsDefault = true);
```

Comment:
not _visible_ or not _editable_?
visible but not editable parameters cannot change as well

Reply:
That should be equivalent in this case. I'll check it out